app/src/main/java/com/hedvig/app/feature/settings/Language.kt

```diff
@@ -16,14 +16,14 @@ enum class Language {
     fun apply(context: Context?): Context? {
         val locale = into()
         return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-            applyV24(context, locale)
+            apply(context, locale)
         } else {
-            applyV21(context, locale)
+            applySingleLocale(context, locale)
         }
     }
 
     @SuppressWarnings("deprecation")
-    private fun applyV21(context: Context?, locale: LocaleWrapper): Context? {
+    private fun applySingleLocale(context: Context?, locale: LocaleWrapper): Context? {
```

Comment:
Are we obligated to always use the deprecated API, regardless of API level?

Reply:
yes, my guess is that if the phone uses this single locale implementation we need to do it the deprecated way. But don't ask me why, in my opinion it is strange that phones with higher api:s use this implementation